clang/lib/DPCT/cmake/libDPCT.cmake

```diff
@@ -32,6 +32,7 @@ macro(build_lib_dpct)
     TypeLocRewriters.cpp
     QueryApiMapping.cpp
     LIBCUAPIMigration.cpp
+    CubRule.cpp
```

Comment:
filename need update to align with other filenames.

Reply:
+1